dbms/src/Storages/DeltaMerge/RowKeyRange.h

```diff
@@ -170,11 +182,6 @@ struct RowKeyValue
         return std::make_shared<DecodedTiKVKey>(prefix + *value);
     }
 
-    bool operator==(const RowKeyValue & v) const
-    {
-        return is_common_handle == v.is_common_handle && (*value) == (*v.value) && int_value == v.int_value;
-    }
-
```

Comment:
Maybe recover these lines and remove `inline bool operator==(const RowKeyValue & a, const RowKeyValue & b)` 